iam-login-service/src/main/java/it/infn/mw/iam/api/account/find/FindAccountController.java

```diff
@@ -44,7 +44,7 @@ public class FindAccountController {
   public static final String FIND_BY_LABEL_RESOURCE = "/iam/account/find/bylabel";
   public static final String FIND_BY_EMAIL_RESOURCE = "/iam/account/find/byemail";
   public static final String FIND_BY_USERNAME_RESOURCE = "/iam/account/find/byusername";
-  public static final String FIND_BY_CERT_SUBJECT_RESOURCE = "/iam/account/find/bycertsubject";
+  public static final String FIND_BY_CERT_SUBJECT_AND_ISSUER_RESOURCE = "/iam/account/find/bycertsubjectandissuer";
```

Comment:
No, simply change the logic and two query parameters for the search, subjectDn and issuerDn are needed (URL-encoded). 
e.g. `http://localhost:8080/iam/account/find/bycertsubjectandissuer?certificateSubject=XXX&certificateIssuer=XXX`
Obviously, as soon as it is approved, the documentation will also have to be modified accordingly.

Reply:
Does this mean that we can't search anymore only by cert subject?

Reply:
Yes, because since subjectDn no longer has the uniqueness constraint the search returns two records (in case one account has two certificates with same subject and different issuer) and thus gives an error.

Reply:
Ok, but searching for subject is useful. What's the problem if I get back two (or more) records? who gives the error?

Reply:
But we would get duplicate records, right? The resulting exception is `IncorrectResultSizeDataAccessException` and comes from spring framework.
However, if we want to keep the search by subject, we can revise the logic.

Reply:
I would expect that most of the times the search will be done using just the cert subject, so that has to work in my opinion. The we can add the possibility to search specifying also the issuer. It wouldn't be bad to search also by serial number and issuer (for example). I wonder if it wouldn't be better to leave /bycertsubject as it is for backwards compatibility (fixing possible bugs, of course) and add a new /bycert, with the new options

Reply:
But... I don't think we keep the serial number, so a search based on it doesn't make much sense, right?

Reply:
Exactly, that's how it is!